src/utils/extractHeight.js

```diff
@@ -9,12 +9,32 @@ export async function extractHeight(ifcModel) {
     const ifcBuildingStoreyID = 3124254112
```

Comment:
It's great you found this!  Much better than guessing/computing from storey height.

However, taking a closer look at this code, I think it should be restructured, see below.  More importantly, not sure this is the algorithm we want.  Let's discuss on chat

```
const IFC_TYPE_ID_BUILDING_STOREY = 3124254112
const IFC_TYPE_ID_SI_UNIT = 448429030
const SI_PREFIX_VALUE_MILLI = 0.001
const SI_PREFIX_VALUE_CENTI = 0.01


export async function extractHeight(ifcModel) {
  const elevValues = []
  const storeys = await ifcModel.getAllItemsOfType(IFC_TYPE_ID_BUILDING_STOREY, true)
  const siUnits = await ifcModel.getAllItemsOfType(IFC_TYPE_ID_SI_UNIT, true)

  let unitScale = 1

  for (let i = 0; i < siUnits.length; i++) {
    if (siUnits[i].UnitType.value === 'LENGTHUNIT') {
      const prefix = siUnits[i].Prefix.value
      switch (prefix) {
        case 'MILLI': unitScale = SI_PREFIX_VALUE_MILLI; break
        case 'CENTI': unitScale = SI_PREFIX_VALUE_CENTI; break
        default: {
          console.warn('Unhandled length unit: ', prefix)
          unitScale = 1
        }
      }
    }
  }

  for (let i = 0; i < storeys.length; i++) {
    const elevation = storeys[i].Elevation.value
    if (!isFinite(elevation)) {
      console.warn('Found invalid elevation value: ', elevation)
      continue
    }
    elevValues[i] = elevation * unitScale
  }

  return elevValues
}
```

Reply:
Ah yeah, you should restore the magic numbers for now, for IFCBUILDINGSTOREY and IFCSIUNIT.  It really should use an import to get those as symbols, but can handle that later

Reply:
Done